aplus/en/formats.py

```diff
@@ -1,2 +1,6 @@
 
-DATETIME_FORMAT = 'D, M j Y, P'
+DATE_FORMAT = 'j F, Y'
```

Comment:
Should this comma be here or not? I'm not sure which would be better for this shorter format, but it's different compared to the longer date format.

Reply:
It does make sense to remove the comma so it would be similar to the longer format, or then it could be same as the Finnish `DATE_FORMAT`, I'm not sure which is better.

`25 August 2020`
`25.8.2020`

Reply:
I think `25 August 2020` (i.e. `j F Y`) or `25 Aug 2020` (i.e. `j M Y` and en_GB default) would be ok, if we use `26. elokuuta 2020` (i.e. `j. E Y` and fi default) for the fi version.

However, this doesn't matter as neither of these are used based on your analysis.